src/libstd/lib.rs

```diff
@@ -515,31 +510,52 @@ mod std_detect;
 #[cfg(not(test))]
 pub use std_detect::detect;
 
-// Re-export built-in macros from core.
+// Re-export macros defined in libcore.
+#[stable(feature = "rust1", since = "1.0.0")]
+pub use core::{
+    // Stable
+    assert_eq,
+    assert_ne,
+    debug_assert_eq,
+    debug_assert_ne,
+    debug_assert,
+    r#try,
+    unimplemented,
+    unreachable,
+    write,
+    writeln,
+    // Unstable
+    todo,
+    uninit_array,
```

Comment:
Drive-by fix - recently introduced `uninit_array` was defined in core, but not reexported in std.

Reply:
FWIW, `uninit_array` should go away with the next bootstrap bump anyway.

Reply:
You can remove the macro if possible; it is rendered obsolete by https://github.com/rust-lang/rust/issues/49147.

Reply:
Ah, ok.
Will remove.

Reply:
You cannot remove it yet because https://github.com/rust-lang/rust/issues/49147 isn't in the bootstrap compiler yet.

Reply:
The reexport, I mean.